packages/techdocs-common/src/stages/publish/types.ts

```diff
@@ -43,6 +43,7 @@ export type PublishResponse = {
 export type TechDocsMetadata = {
   site_name: string;
   site_description: string;
+  etag: string;
```

Comment:
would be nice with some description of the etag metadata maybe, but will not block on that!

Reply:
Sure! Will add it here as well.